### PR TITLE
[WebSocket] Fix connection closing issue

### PR DESCRIPTION
**Pull request by irunika**
This PR addresses https://github.com/ballerina-platform/ballerina-lang/issues/7893.

Furthermore, there are few more steps to fully support WebSocket connection closure and will be available in upcoming releases.

Resolve https://github.com/ballerina-platform/ballerina-lang/issues/7821

### CODE REVIEW

File: stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/WebSocketDispatcher.java [hunk 1]
```diff
@@ -187,8 +210,25 @@ public static void dispatchCloseMessage(WebSocketOpenConnectionInfo connectionIn
         bValues[1] = new BInteger(closeMessage.getCloseCode());
```
Comment (riyafa) [inline stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/WebSocketDispatcher.java]: Shall we handle the case where close status code is -1? We cannot pass it to the resource. It should be 1006 if it is an abnormal closure.

File: pom.xml [hunk 2]
```diff
@@ -1553,7 +1553,7 @@
         <equinox.osgi.version>3.10.2.v20150203-1939</equinox.osgi.version>
         <equinox.osgi.services.version>3.4.0.v20140312-2051</equinox.osgi.services.version>
         <carbon.kernel.version>5.1.0</carbon.kernel.version>
-        <transport.http.version>6.0.156</transport.http.version>
+        <transport.http.version>6.0.158</transport.http.version>
```
Comment (arukshani) [inline pom.xml]: transport was updated to 159. we dont need this change.

Comment (irunika) [reply pom.xml]: Done

File: stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/WebSocketDispatcher.java (hunk 1 above)
Comment (irunika) [reply stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/WebSocketDispatcher.java]: done https://github.com/wso2/transport-http/pull/148

**Pull request merged**